### DISCUSSION TITLE
voice response

### DISCUSSION THREAD

**Discussion by bshass** [JarvyJ/HomeIntent#393] [Support Questions] [answered]
not very knowledgeable regarding programming in general and really struggled getting rhasspy to integrate with HA (never really got it to work).  But HomeIntent was significantly easier (once i learned how docker works), and seems to work right out of the box.  so really liking it so far.

one issue.  i have a device called "bedroom light", and when i call for jarvis to turn it on/off, the voice response says "Turning on the bedroom light light".  Sounds like it's adding a second "light" as a suffix to the device name, when it's not necessary.

**Comment by JarvyJ** [comment]
Oh yeah, I can see that being an issue. I'll add some code to see if the last word is repeated and omit it necessary. I'll create an issue for it.

A workaround if you need something right away would be renaming the entity in Home Assistant.

**Reply by bshass** [reply]
yup, not a huge deal just wanted to give a heads up. also something else im going to create another thread for

**Comment by JarvyJ** [comment, marked as answer]
The PR is #395 and will make it into the next release! Thanks for letting me know
